fixtures/small/private_def_expected.rb

```diff
@@ -0,0 +1,5 @@
+class Foo
+  private def bar
```

Comment:
@penelopezone To double check before I dive in, is this the expected output here? I wasn't too clear when we chatted earlier.